vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -4016,6 +4013,22 @@ public void onClick(DialogInterface dialog, int which) {
             }
         }
     }
-}
 
+    /**
+     * Action when add a room name on room creation.
+     */
+    private void addRoomName() {
+        // sanity check
+        if ((null == mRoom) || (null == mSession)  || (null == mDefaultRoomName)) {
+            return;
+        }
+
+        // update only, if values is not empty
+        if (!mDefaultRoomName.isEmpty()) {
+            showWaitingView();
 
+            Log.d(LOG_TAG, "##addRoomName to " + mDefaultRoomName);
+            mRoom.updateName(mDefaultRoomName, null);
```

Comment:
It would be better to define one method to handle mRoom.updateName call in this activity